api/v1alpha1/selfnoderemediationconfig_webhook.go

```diff
@@ -172,3 +175,26 @@ func validateToleration(toleration v1.Toleration) error {
 	}
 	return nil
 }
+
+func (r *SelfNodeRemediationConfig) validateMinRebootTime() error {
+	if r.Status.MinSafeTimeToAssumeNodeRebootedSeconds == 0 {
+		return fmt.Errorf("failed to verify min value of SafeRebootTimeSec, Status.MinSafeTimeToAssumeNodeRebootedSeconds should not be empty")
+	}
+
+	//allow removing this optional field
+	if r.Spec.SafeTimeToAssumeNodeRebootedSeconds == 0 {
```

Comment:
I agree, I'm just not sure regarding whether this qualifies as an API change or not :thinking: , WDTY ?

Reply:
I wouldn't consider making a field optional as API change. The other way around I would.

Reply:
Technically I don't think we are making the field optional since IIRC it's optional by default in case it doesn't have `required`, we just remove the default value  (In any case I agree we are in the clear).

What I was worried of is changing from `int` to `*int`.
IIUC you don't think it's an issue ?